core/trino-main/src/main/java/io/trino/execution/SqlTask.java

```diff
@@ -90,14 +90,14 @@
     private final AtomicReference<TaskHolder> taskHolderReference = new AtomicReference<>(new TaskHolder());
     private final AtomicBoolean needsPlan = new AtomicBoolean(true);
 
-    public static SqlTask createSqlTask(
+    static SqlTask createSqlTask(
```

Comment:
unrelated?

Reply:
It's one of the methods that gets the API changed, I just wanted to hide it somewhat to make sure that the impact is limited. Should I leave it `public`?

Reply:
The public or not is orthogonal to this PR, so keep it as it was.

BTW trino-main has no "public API", it's not a library

Reply:
Sure, I backed out this change.